src/home_robot/home_robot/navigation_planner/discrete_planner.py

```diff
@@ -167,26 +167,24 @@ def plan(
                     self.curr_obs_dilation_selem_radius
                 )
 
+        stg_x, stg_y = short_term_goal
+        angle_st_goal = math.degrees(math.atan2(stg_x - start[0], stg_y - start[1]))
+        angle_agent = start_o % 360.0
+        if angle_agent > 180:
+            angle_agent -= 360
+
+        relative_angle = (angle_agent - angle_st_goal) % 360.0
+        if relative_angle > 180:
+            relative_angle -= 360
```

Comment:
Make this angle normalization logic into a helper function, ideally within `home_robot.utils`.